integration-tests/main/src/main/java/io/quarkus/example/rest/TestResourceWithConstructorInjection.java

```diff
@@ -0,0 +1,20 @@
+package io.quarkus.example.rest;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+
+@Path("/testCtor")
+public class TestResourceWithConstructorInjection {
+
+    final Someservice service;
+
+    public TestResourceWithConstructorInjection(Someservice service) {
```

Comment:
@geoand could you add a test with a class having a default constructor and constructor injection too so that we can check it works OK?

Reply:
Of course, on it

Reply:
Fixed